plugins/learneth/profile.json

```diff
@@ -0,0 +1,12 @@
+{
+ "name": "LearnEth",
+ "displayName": "LearnEth",
+ "description": "Learn Ethereum with Remix !",
```

Comment:
Remix ! -> Remix!  ( you have an extra space)

Reply:
done